src/Text/Pandoc/Writers/Powerpoint/Output.hs

```diff
@@ -173,6 +174,10 @@ getSlideLayouts = asks envSlideLayouts >>= maybe (throwError e) pure
 data ContentType = NormalContent
                  | TwoColumnLeftContent
                  | TwoColumnRightContent
+                 | ComparisonUpperLeftContent
+                 | ComparisonUpperRightContent
+                 | ComparisonLowerLeftContent
+                 | ComparisonLowerRightContent
```

Comment:
I'm not mad about this, because basically I'll need a new constructor for each placeholder in every supported layout. Maybe a better choice would be to specify the placeholder type and index here?

```
newtype ContentType 
  = ContentType PHType Int
  -- ^ The placeholder type to use for the content, 
  -- and the index of the placeholder to use.
```

Reply:
Ok, I think I will – thanks